### ISSUE TITLE
Remove Gopls from setup in ".devcontainer" since it's now enabled in VS Code by default.

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/Hello-Cobra#14]
## TL; DR

> Remove `Gopls` form the Dockerfile for `.devcontainer`

https://github.com/KEINOS/Hello-Cobra/blob/81a3c24d75295547cf115c3971d92555fee9e0b7/.devcontainer/Dockerfile#L62-L65

## TS; DR

As of `Go for VS Code` v0.22.0, the  _VS Code extension will enable `Gopls` by default_.

It still works as is now, and we **need to concern the side effects to "Codespaces"**. But why not let the VS Code do it for us?

- "[Enable `gopls` by default](https://github.com/golang/vscode-go/issues/1037)" | VSCode Go | Golang @ GitHub

<kbd>![スクリーンショット 2021-01-28 9 29 04](https://user-images.githubusercontent.com/11840938/106072437-8c58bf80-614b-11eb-9fb6-043fd1b1ca18.png)</kbd>

Here's the list of tools that will be installed by default when the extension gets installed.

- [Tools](https://github.com/golang/vscode-go/blob/master/docs/tools.md) | Docs | VS Code Go | Golang @ GitHub

**Comment by KEINOS** [comment]
## May not be needed to remove

I confirm that installing the `Go` extension of VSCode will activate `gopls`. But the version of `gopls` in the base image `mcr.microsoft.com/vscode/devcontainers/go` of the container is v0.6.4. Which is an older version.

So leaving as is and let `go get -u -v` update the `gopls` package seems to be fine.
